internal/driver/driver_test.go

```diff
@@ -63,6 +63,8 @@ func TestParse(t *testing.T) {
 		{"text,lines,cum,show=[12]00", "cpu"},
 		{"text,lines,cum,hide=line[X3]0,focus=[12]00", "cpu"},
 		{"topproto,lines,cum,hide=mangled[X3]0", "cpu"},
+		{"topproto,lines", "cpu_repeated_function"},
```

Comment:
Can we extend one of existing cases to have a repeated function? It doesn't seem the case with a function referenced by several locations needs to be special and having so many almost-the-same test cases becomes messy, let's try to simplify things.

Reply:
Done.
Did more testing on master, and current cpuProfile() has repeated functions, so can be used for this.